tests/test_explorer.py

```diff
@@ -7,11 +7,12 @@
 import pytest
 from context import Explorer, ut
 
+from fmu.sumo.explorer._document_collection import DocumentCollection
```

Comment:
The point with the import from context is that you can run without having to pip install the environment. Importing directly here brakes the pattern. I am not saying we need to follow the pattern, just that we should be consistent. Maybe come up with a common way of setting up tests. Using conftest.py is another way I have learned from reviewing Jans code..

Reply:
Yes, good point, I have no strong feelings about that. I can change it to import from context.

Reply:
Changed it now. I don't think it matters _where_ the import statement is, as long as local files are inserted into sys.path (which is what the context.py is doing).